packages/strapi-design-system/src/SimpleMenu/SimpleMenu.js

```diff
@@ -0,0 +1,111 @@
+import React, { useRef, useState, Children, cloneElement } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import FilterDropdown from '@strapi/icons/FilterDropdown';
+import { TextButton, Text } from '../Text';
+import { Box } from '../Box';
+import { Row } from '../Row';
+import { Popover } from '../Popover';
+import { getOptionStyle } from './utils';
+
+const OptionButton = styled.button`
+  border: none;
+  padding: 0;
+  background: transparent;
+  cursor: pointer;
+  ${getOptionStyle}
+`;
+const OptionLink = styled.a`
+  text-decoration: none;
+  ${getOptionStyle}
+`;
+const MenuButton = styled.button`
+  border: none;
+  background: transparent;
+  display: flex;
+  align-items: center;
+  font-size: ${12 / 16}rem;
+  svg {
+    height: 4px;
+  }
+`;
+
+export const MenuItem = ({ children, onClick, href, ...props }) => {
```

Comment:
I think that the MenuItems should have a `tabIndex={-1}`. 

**Why?**

`tabIndex={-1}` aims to make an element focusable programmatically. While it's not possible to target them using the `tab` key, it's still possible to focus them with `element.focus()`.

**Why do we want that?**

Generally, when providing an experience to people using the keyboard, the `tab` key aims to get to the next "widget" (or component, or "compound group of elements"). In the actual state of the component, using tab gets you to the next children of the menu (e.g the MenuItem). I do think we can have a better experience by preventing tabs from getting to the next menuitem, but instead to get the focus to the next focusable widget.

For instance, imagine that we have 20 elements in the menu list: do we want the user to tab 21 times to get out of the box and focus on the next element? 😊 

**How to access the next menuitem then?**

This is where keys enter the game: using arrow up, arrow down and eventually addition keys (home + arrow left etc...)

Let me know If this is not clear enough 😊

Reply:
It is super clear thank you. I was wondering that also but I wasn't sure about it.